daemons/gptp/common/ieee1588port.cpp

```diff
@@ -617,13 +618,17 @@ void IEEE1588Port::processEvent(Event e)
 					}
 					if (EBest == NULL) {
 						EBest = ports[j]->calculateERBest();
```

Comment:
This could still be `NULL` afterward if the port hasn't received an announce yet and `calculateERBest()` returns `NULL` at this point.